pkg/controller/lifecycle/actuator.go

```diff
@@ -130,29 +133,56 @@ func (a *actuator) Reconcile(ctx context.Context, ex *extensionsv1alpha1.Extensi
 		return err
 	}
 
-	genericKubeconfigName := extensions.GenericTokenKubeconfigSecretNameFromCluster(cluster)
-	seedResources, err := getSeedResources(oidcReplicas, hibernated, namespace, genericKubeconfigName, oidcShootAccessSecret)
+	// initialize SecretsManager based on Cluster object
+	var (
+		caName        = "ca-extension-shoot-oidc-service"
+		secretConfigs = []extensionssecretsmanager.SecretConfigWithOptions{
+			{
+				Config: &secretutils.CertificateSecretConfig{
+					Name:       caName,
+					CommonName: caName,
+					CertType:   secretutils.CACert,
+				},
+				Options: []secretsmanager.GenerateOption{secretsmanager.Persist()},
+			},
+			{
+				Config: &secretutils.CertificateSecretConfig{
+					Name:       constants.WebhookTLSecretName,
```

Comment:
Typo: `WebhookTLSSecretName`

Reply:
Thanks! I can rename this in a separate PR so we can keep the changes to a minimum for this one

Reply:
No problem, it's rather straight-forward now, so I add it to my "address PR review" commit